contracts/deposit-contract/depositContract_test.go

```diff
@@ -110,45 +109,6 @@ func TestValidatorRegister_OK(t *testing.T) {
 	}
 }
 
-// normal test case, test beacon chain start log event.
-func TestETH2Genesis_OK(t *testing.T) {
```

Comment:
same here. i just name change and keep the test. just skip it

Reply:
This is part of the deposit contract tests, so makes no sense to test for genesis trigger over here, since chainstart is handled by the node now.